packages/web/pages/user/my-account/my-services.js

```diff
@@ -48,7 +48,6 @@ const MyServices = ({
 				totalPages: initialTotalPages,
 				totalItems: initialTotalItems,
 			},
-			revalidateOnMount: true,
```

Comment:
This was triggering a duplicated request.

We don't need o `revalidateOnMount` as we're passing data from `getInitialProps`